src/text-editor.coffee

```diff
@@ -1948,10 +1948,11 @@ class TextEditor extends Model
   # This method may merge selections that end up intesecting.
   #
   # * `position` An instance of {Point}, with a given `row` and `column`.
```

Comment:
I'm not sure I want to make it part of the public API, since it leaves us in a weird state where selections overlap.